src/AuthManager.ts

```diff
@@ -174,6 +174,8 @@ export class AuthManager {
         return new Promise((resolve, reject) => {
             try {
                 const bearerToken = localStorage.getItem('access_token');
+                localStorage.removeItem('access_token');
+                localStorage.removeItem('refresh_token');
```

Comment:
I think that @SirNarsh meant that Should be this at the `catch` block as well! 🤔 
Or, did I get this wrongly...?

Reply:
@MostafaFawzy7 That's valid. But I think the line 176, won't throw an error.